code/atom.dm

```diff
@@ -304,6 +304,10 @@ TYPEINFO(/atom)
 /atom/proc/ex_act(var/severity=0,var/last_touched=0, var/power=0, var/datum/explosion/explosion=null)
 	return
 
+/// Use this proc to check if you can use and permanently change this item via a combination
+/atom/proc/is_combination_consumeable()
+	return TRUE
```

Comment:
What is a "combination"? I don't really know what this is checking and think this is a poorly named proc.

(Also: consumable, not consumeable)